### DISCUSSION TITLE
New placement of site setting button is terrible.

### DISCUSSION THREAD

**Discussion by quarzasiphix** [elementor/elementor#28498] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

I often click the site setting button accidently, 
The button being between structure and add elements is a terrible idea imo,
site setting is a button I use sometimes, 
and I click between structure and add element every few seconds.

And there is no way to cancel going into site settings when you accidently click it and you have to wait till it loads fully and then click go back and wait to load in again.

### Describe the solution you'd like.

Maybe move the button somewhere to side for site settings. 

### Describe alternatives you've considered.

Moving the placement of site settings button 

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by avs-alexandra** [comment]
+1

**Comment by MarcCeleiro** [comment]
+1, and I'm disabling the new bar in all projects because it's extremely confusing.
